### PR TITLE
jsonnet/kube-prometheus: Add back kube-rbac-proxy containers to

### PR DESCRIPTION
**Pull request by lilic**
kube-state-metrics. These were removed by accident when migrating to
using upstream libsonnet.

Still need to test it out. But ready for review. @coreos/team-monitoring

### CODE REVIEW

**Comment by s-urbaniak** [comment]
lgtm

**Comment by lilic** [comment]
WIP: need to solve a few more things.

**Comment by lilic** [comment]
@s-urbaniak ready for re-review @brancz @paulfantom @pgier

File: Makefile [hunk 1]
```diff
@@ -63,4 +63,4 @@ test-in-docker:
 .PHONY: generate generate-in-docker test test-in-docker fmt
 
 $(JB_BINARY):
-	go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+	go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@v0.2.0
```
Comment (s-urbaniak) [inline Makefile]: :pray:

File: manifests/kube-state-metrics-service.yaml [hunk 2]
```diff
@@ -9,11 +9,11 @@ metadata:
...
-    targetPort: http-metrics
-  - name: telemetry
-    port: 8081
-    targetPort: telemetry
+  - name: https-main
+    port: 8443
+    targetPort: https-main
+  - name: https-self
+    port: 9443
+    targetPort: https-self
```
Comment (s-urbaniak) [inline manifests/kube-state-metrics-service.yaml]: do these port changes matter to us?

Comment (lilic) [reply manifests/kube-state-metrics-service.yaml]: Yes, we had this before merging the kube-state-metrics PR that introduced using the upstream kube-state-metrics libsonnet.

 See -> https://github.com/coreos/kube-prometheus/blob/release-0.4/manifests/kube-state-metrics-service.yaml

**Comment by paulfantom** [comment]
lgtm :shipit:

**Comment by brancz** [comment]
Assuming this was tested, lgtm :+1:

**Comment by lilic** [comment]
Yes tested and can see no errors in list/watch of kube-state-metrics resources and its the targets are up otherwise the tests would not pass:
> monitoring/kube-state-metrics/0 (1/1 up)
> monitoring/kube-state-metrics/1 (1/1 up)

**Pull request merged**